src/app/core/services/config/remote-config.service.ts

```diff
@@ -34,14 +36,15 @@ export interface RemoteConfig {
 class FirebaseRemoteConfig implements RemoteConfig {
   constructor(private firebase: Firebase, private logger: LogService) {}
 
-  get(key: ConfigKeys): Promise<string | null> {
-    return this.firebase.getValue(key.value, '')
+  get(key: ConfigKeys) {
```

Comment:
@blootsvoets I completely forgot about the deprecated FirebaseInstanceID and namespace params in the firebase plugin (https://firebase.google.com/support/release-notes/android#version_1650). 
I previously made the changes here which was reverted in the new version: 
https://github.com/RADAR-base/cordova-plugin-firebase/commit/0649f7b990e4cde1808ebadd3edfe90e1fb103dc
https://github.com/RADAR-base/cordova-plugin-firebase/commit/b0f5c1c460721023838e2d4479930c59615c249c

So the ionic native version has not been updated yet. We can use the FirebasePlugin variable to use the updated methods and keep the old version of `cordova-plugin-firebase-with-upstream-messaging`.

Reply:
@mpgxvii The namespace params are deprecated, but can still be used (optional). They are also still used in the `ionic-native` package. Without the namespace param, the `getValue` method does not work, because the order of the variables is wrong.

I propose to use `cordova-plugin-firebase-with-upstream-messaging` 2.1.2 so that the namespace can OPTIONALLY be set and the plugin works with `ionic-native`. If you want to remove the firebase instance ID, can you please remove it in the next version of `cordova-plugin-firebase-with-upstream-messaging`?

Reply:
FYI: the cordova plugin still expects two variables to the firebase plugin: https://github.com/ionic-team/ionic-native/blob/f6e77db7b06f391fe31e4367101cbc9d2fccb369/src/%40ionic-native/plugins/firebase/index.ts#L218.

Reply:
I changed it because I was getting this error during the build: 
`apache/cordova/firebase/FirebasePlugin.java:677: error: method getByteArray in class FirebaseRemoteConfig cannot be applied to given types;
                        : FirebaseRemoteConfig.getInstance().getByteArray(key, namespace);
                                                            ^
  required: String
  found: String,String`

The `ionic-native` plugin seems to be unmaintained so I overrode it using `FirebasePlugin.getValue` instead of `this.firebase.getValue`

I wasn't sure how to resolve this error without using the fix (https://github.com/RADAR-base/cordova-plugin-firebase/commit/0649f7b990e4cde1808ebadd3edfe90e1fb103dc and `FirebasePlugin.getValue`)

Reply:
OK, I'll revert in the PR.